recipes/configure_server.rb

```diff
@@ -142,6 +145,11 @@
                    end
 end
 
+# Reload ohai for version info on packages before creating the cluster
+ohai 'reload' do
+  action :reload
+end
+
```

Comment:
This resource name seems generic and a fair chance of naming overlap.  Should this be a unique name like `reload_percona`?